### ISSUE TITLE
Collection of interactions input file 9606.protein.links.full_w_homology

### ISSUE THREAD

**Issue by mbdebian** [opentargets/issues#2818]
## Background
The following file is not being collected by PIS automatically
> interactions-inputs/9606.protein.links.full_w_homology.v11.5.txt.gz

This file is an enrichment produced by the @opentargets/data-team on top of the original source data file.

We need a location for PIS where to grab the latest version of this as part of the _interactions_ data collection step.

**Comment by mbdebian** [comment]
@DSuveges @ireneisdoomed , I think this file could be in a google bucket location where PIS can automatically select the latest version of the file.

Right now, PIS supports versioning based on timestamps, but using a name like
> 9606.protein.links.full_w_homology._\<timestamp\>_.txt.gz

would probably loose the information related to the original source's file that was used to produce this one.
Please, let me have a look into this specific part of PIS to find out whether there is an easy way of preserving that information.

**Comment by mbdebian** [comment]
As this files hardly ever changes, and we end up copying over the one we used in the previous release, this has been set to be collected by PIS from our EBI data repository at
> https://ftp.ebi.ac.uk/pub/databases/opentargets/platform/22.11/input/interactions-inputs/9606.protein.links.full_w_homology.v11.5.txt.gz

With destination filename
> interactions-inputs/9606_protein_links.txt.gz

As expected by the ETL, according to its configuration file
```scala
  strings {
    format = "csv"
    path = ${common.input}"/interactions-inputs/9606_protein_links.txt.gz"
    options = [
      {k: "sep", v: " "},
      {k: "header", v: "true"}
    ]
  }
```
Although I'm not sure the file name is of any importance, if we have a look at the previous release _input/interactions-inputs_ folder, there is no _9606\_protein\_links.txt.gz_, the original file name, _9606.protein.links.full\_w\_homology.v11.5.txt.gz_ was kept.
> gsutil ls -alh gs://open-targets-data-releases/22.11/input/interactions-inputs

```
106.27 MiB  2022-11-24T08:44:53Z  gs://open-targets-data-releases/22.11/input/interactions-inputs/9606.protein.links.full_w_homology.v11.5.txt.gz#1669279493668945  metageneration=1
 35.88 MiB  2022-11-24T08:44:54Z  gs://open-targets-data-releases/22.11/input/interactions-inputs/HUMAN_9606_idmapping.dat.gz#1669279494403026  metageneration=1
  51.6 MiB  2022-11-24T08:44:53Z  gs://open-targets-data-releases/22.11/input/interactions-inputs/Homo_sapiens.GRCh38.chr.gtf.gz#1669279493805309  metageneration=1
777.46 MiB  2022-11-24T08:45:03Z  gs://open-targets-data-releases/22.11/input/interactions-inputs/intact-interactors.json#1669279503571330  metageneration=1
109.77 MiB  2022-11-24T08:44:54Z  gs://open-targets-data-releases/22.11/input/interactions-inputs/rna_central_ensembl.tsv#1669279494952359  metageneration=1
110.19 MiB  2022-11-24T08:44:55Z  gs://open-targets-data-releases/22.11/input/interactions-inputs/string-interactions.json.gz#1669279495679819  metageneration=1
TOTAL: 6 objects, 1249033768 bytes (1.16 GiB)
```

**Comment by d0choa** [comment]
For reference, the data comes from the STRING database.

As today we keep in the 11.5 version which has been the public release for quite a while. More info in:
https://string-db.org/cgi/download

Due to incompleteness in the data (String bug?), @DSuveges patched their previous release data (#1509). We didn't have enough incentive to make this fully automatic, so we keep this static patched data as our input. Next time there is a release we will have to check if the issue is still there and potentially patch the data.

**Comment by mbdebian** [comment]
This is now brought forward from previous releases, and automatically collected by PIS from our EBI FTP repository as part of the _interactions_ step.
